examples/tutorials/flow-deploy/flow-model-packaging/README.md

```diff
@@ -0,0 +1,161 @@
+# Flow model packaging
+This example demos how to package flow as a executable app. 
+We will use [web-classification](../../../flows/standard/web-classification/README.md) as example in this tutorial.
+
+Please ensure that you have installed all the required dependencies. You can refer to the "Prerequisites" section in the README of the [web-classification](../../../flows/standard/web-classification/README.md#Prerequisites) for a comprehensive list of prerequisites and installation instructions.
+
+## Build a flow as docker format
+
+Note that all dependent connections must be created before building as docker.
+```bash
+# create connection if not created before
+pf connection create --file ../../../connections/azure_openai.yml --set api_key=<your_api_key> api_base=<your_api_base> --name open_ai_connection
+```
+
+Use the command below to build a flow as docker format app:
+
+```bash
+pf flow build --source ../../../flows/standard/web-classification --output target --format docker
+```
+## Package flow model
+### Prepare an entry file
+A Python entry file is included as the entry point for the bundled app. We offer a Python file named `start.py`` here, which enables you to serve a flow folder as an endpoint.
+
+```python
+import subprocess
+import os
+import json
+
+def setup_promptflow(requirement_path) -> None:
+    if os.path.exists(requirement_path):
+        print("- Setting up the promptflow requirements")
```

Comment:
why do we need to do this? I thought the pyinstaller will do this for us?

Reply:
updated